.github/renovate.json

```diff
@@ -41,6 +41,12 @@
         "// renovate: datasource=(?<datasource>[a-z-]+?) depName=(?<depName>.+?)(?: versioning=(?<versioning>[a-z-]+?))?\\s+readonly\\s+tag\\s+=\\s+'(?<currentValue>.+?)';"
       ],
       "versioningTemplate": "{{#if versioning}}{{versioning}}{{else}}semver{{/if}}"
+    },
+    {
```

Comment:
Do we also need to explicitly enable the `regex` manager for this or is that enabled by default? I couldn't find this in the docs.

Reply:
nope, is enabled by default.